backend/app/api/sso.py

```
@@ -248,7 +270,9 @@ async def get_saml_metadata(tenant_slug: str, db: Session = Depends(get_db)):
 
 
 @router.get("/.well-known/openid_configuration/{tenant_slug}")
-async def get_oidc_configuration(tenant_slug: str, db: Session = Depends(get_db)):
+async def get_oidc_configuration(
+    tenant_slug: str, db: Session = Depends(get_db)
```

Ruff (0.12.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)